web/src/main/java/org/mskcc/cbio/portal/web/ProxyController.java

```diff
@@ -65,7 +65,7 @@
   public void setHotspotsURL(String property) { this.hotspotsURL = property; }
   
   private String bitlyURL;
-  @Value("${bitly.url}")
+  @Value("${bitly.url:''}")
```

Comment:
why is this still here if you're getting rid of the property?

Reply:
I would rather get leave things be and then retire it all en-masse

Reply:
i'm not quite taking it out.    i want to remove it from EXAMPLE property file because it actually is defunct and does nothing.  but the property needs to have a default value or the above code will cause compilation error.  so i think this is the lightest touch.

Reply:
ah okay